recipe-hooks-project/src/AppPage.test.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import {cleanup, fireEvent, render} from "@testing-library/react";
```

Comment:
You could add `waitForElement` to the imports from this file (and remove it from `@testing-library/dom`)